Homework 2/SalesRepresentativeOrder/src/main/java/ro/siit/BubbleSort.java

```diff
@@ -0,0 +1,30 @@
+package ro.siit;
+
+import static ro.siit.SalesRepresentativeOrder.salesRep;
```

Comment:
You are statically importing the internal structure of a different class. Think about OOP encapsulation rules for future homeworks. In this homework it was out of scope, so it is not an issue that needs to be fixed :)